nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/DataBucket.java

```diff
@@ -41,18 +41,32 @@
 
     protected byte[] data;
 
+    protected byte[] header;
+
+    protected byte[] footer;
```

Comment:
I didn't follow this class, isn't it something too specific for this class ?

Reply:
well the pattern is common we split something to have a distribtued processing, then we want to merge it and still have a header and footer, just think about a json or xml export it will be the same need.